src/main/java/com/cloudbees/jenkins/support/SupportCommand.java

```diff
@@ -107,7 +98,7 @@ protected int run() throws Exception {
         }
 
         public OutputStream call() throws IOException {
-            File f = File.createTempFile(filename, ".zip");
+            File f = File.createTempFile(filename, "");
```

Comment:
Though if we are always adding the current date down to second resolution anyway, then perhaps we can omit the generated suffix and just save `${java.io.tmpdir}/${filename}`.